README.md

```diff
@@ -19,60 +19,120 @@ GeoMesa is an open-source, distributed, spatio-temporal database built on top of
 
-<b>NOTE:</b> The current recommended version is ```1.1.0-rc1```, which includes breaking changes from ```1.0.0-rc7``` and previous versions due to schema incompatibility and changes in indexing structure. The most recent tar.gz assembly can be 
-[downloaded here](http://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-assemble/1.1.0-rc.1/geomesa-assemble-1.1.0-rc.1-bin.tar.gz) which contains the [Accumulo distributed runtime jar](geomesa-distributed-runtime), [GeoServer plugin](geomesa-plugin), and [command line tools](geomesa-tools).
+<b>NOTE:</b> The current recommended version is ```1.1.0-rc.6```, which includes breaking changes from ```1.0.0-rc.7``` and previous versions due to schema incompatibility and changes in indexing structure. The most recent tar.gz assembly can be 
+[downloaded here](http://repo.locationtech.org/content/repositories/geomesa-releases/org/locationtech/geomesa/geomesa-assemble/1.1.0-rc.6/geomesa-assemble-1.1.0-rc.1-bin.tar.gz) which contains the [Accumulo distributed runtime jar](geomesa-distributed-runtime), [GeoServer plugin](geomesa-plugin), and [command line tools](geomesa-tools).
 
 GeoMesa artifacts can be downloaded from the [LocationTech Maven repository](https://repo.locationtech.org/content/repositories/geomesa-releases/)
 
 Snapshots are available in the [LocationTech Snapshots Repository](https://repo.locationtech.org/content/repositories/geomesa-snapshots/)
 
 
-## Building Instructions
+## Building
 
-* Navigate to where you would like to download this project.
-* ```git clone git@github.com:locationtech/geomesa.git```
+Navigate to where you would like to download this project.
 
-This project is managed by Maven, and builds with the command
+    git clone git@github.com:locationtech/geomesa.git
+    cd geomesa
+    build/mvn clean install
 
-``` geomesa> mvn clean install ```
-
-From the root directory, this builds each sub-project with its additional dependencies-included JAR.
+This project is managed by Maven, and builds using Maven with [Zinc](https://github.com/typesafehub/zinc).
+From the root directory, the above will build each sub-project with its additional dependencies-included JAR.
 
 ## Documentation
 
-* [Quick Start](http://geomesa.github.io/#quick-start) on the main [documentation](http://geomesa.github.io) site.
-* [FAQ](http://geomesa.github.io/faq/)
-* [Tutorials](http://geomesa.github.io)
+* [Quick Start](http://www.geomesa.org/geomesa-quickstart/) on the main [documentation](http://www.geomesa.org/) site.
+* [FAQ](http://www.geomesa.org/faq/)
+* [Tutorials](http://www.geomesa.org/tutorials/)
 * GeoMesa [Users](https://locationtech.org/mhonarc/lists/geomesa-users/) and [Dev](https://locationtech.org/mhonarc/lists/geomesa-dev/) mailing list archives
 * READMEs are provided under most modules: [Tools](geomesa-tools), [Jobs](geomesa-jobs), etc
 
 ## GeoMesa Project Structure
 
-#### geomesa-core
+#### geomesa-accumulo
```

Comment:
Maybe change it from headlines to just a bulleted list? With a single line per module like anthony says. Module name cam link to readme when there is one.